activiti-cloud-starter-audit/src/test/java/org/activiti/cloud/starter/audit/tests/it/AuditServiceIT.java

```diff
@@ -582,6 +586,74 @@ public void checkEventConvertersForTaskCandidates() {
         
     }
     
+    @Test
+    public void checkEventConvertersForSignal() {
+        //given
+        ProcessInstanceImpl processInstanceStarted = new ProcessInstanceImpl();
+        processInstanceStarted.setId("processInstanceId");
+        processInstanceStarted.setProcessDefinitionId("processDefinitionId");
+        processInstanceStarted.setProcessDefinitionKey("processDefinitionKey");
+        processInstanceStarted.setBusinessKey("businessKey");
+        processInstanceStarted.setParentId("parentId");
+        
+        
+        BPMNSignalImpl signal = new BPMNSignalImpl("entityId");
+        signal.setProcessDefinitionId(processInstanceStarted.getProcessDefinitionId());
+        signal.setProcessInstanceId(processInstanceStarted.getId());
+
+        SignalPayload signalPayload = ProcessPayloadBuilder.signal()
+                .withName("SignalName")
+                .build();
+        signal.setSignalPayload(signalPayload);
+
+        CloudBPMNSignalReceivedEventImpl event = new CloudBPMNSignalReceivedEventImpl("eventId",
+                                                                                      System.currentTimeMillis(),
+                                                                                      signal,
+                                                                                      signal.getProcessDefinitionId(),
+                                                                                      signal.getProcessInstanceId());
+
+        //Set explicitly to be sure
+        event.setEntityId("entityId");
+        event.setProcessInstanceId(processInstanceStarted.getId());
+        event.setProcessDefinitionId(processInstanceStarted.getProcessDefinitionId());
+        event.setProcessDefinitionKey(processInstanceStarted.getProcessDefinitionKey());
+        event.setBusinessKey(processInstanceStarted.getBusinessKey());
+        event.setParentProcessInstanceId(processInstanceStarted.getParentId());
+        event.setMessageId("message-id");
+        event.setSequenceNumber(0);
+
+        
+             
+        //Check convertToEntity
+        EventToEntityConverter converter = eventConverters.getConverterByEventTypeName(event.getEventType().name());
```

Comment:
This part makes sense to have as integration tests, but I'd rather create a test under `APIEventToEntityConvertersIT`, where you can have one test per eventType.

Reply:
@erdemedeiros 
Here nothing yet made, not clear what exactly to cover in the test

Reply:
A test is added